spec/compiler/crystal/tools/implementations_spec.cr

```diff
@@ -25,7 +25,7 @@ private def assert_implementations(code)
     end
   end
 
-  code = code.gsub('‸', "").gsub('༓', "")
+  code = code.delete('‸').delete('༓')
```

Comment:
`code.delete { |c|  {'‸', '༓'}.includes? c }` may yield to more performance.

Reply:
Yep it's 2 times faster.